This cannot work with wayland yet since it
directly uses fl_display.

diff --git a/src/UI/MiscGui.cpp b/src/UI/MiscGui.cpp
index 10df075..7ec0afb 100644
--- a/src/UI/MiscGui.cpp
+++ b/src/UI/MiscGui.cpp
@@ -28,6 +28,7 @@
 
 #include <FL/Fl.H>
 #include <FL/fl_draw.H>
+#include <FL/platform.H>
 
 #include <cairo.h>
 #include <cairo-xlib.h>
diff --git a/src/UI/WidgetPDial.cpp b/src/UI/WidgetPDial.cpp
index 0839f1f..6f42767 100644
--- a/src/UI/WidgetPDial.cpp
+++ b/src/UI/WidgetPDial.cpp
@@ -32,6 +32,7 @@
 #include <FL/fl_draw.H>
 #include <FL/Fl_Tooltip.H>
 #include <FL/Fl_Group.H>
+#include <FL/platform.H>
 #include <FL/x.H>
 #include <cairo.h>
 #include <cairo-xlib.h>
diff --git a/src/main.cpp b/src/main.cpp
index 100348c..f324a86 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -182,6 +182,8 @@ string runShellCommand(string command)
  */
 int main(int argc, char *argv[])
 {
+    setenv("FLTK_BACKEND", "x11", 1);
+
     /*
      * The following is a way to quickly identify and read key config startup values
      * before the synth engine has started, or any of the normal functions have been
